test/integration/tasks/create-configuration-files.js

```diff
@@ -4,18 +4,21 @@ const printStepTitle = require('../utils').printStepTitle
 
 module.exports = async ({ cmaToken, spaceId, environmentId }) => {
   printStepTitle('Creating configuration files based on environment variables')
+  async function writeJSONForExtension(extensionId) {
+    await writeJSONFile(resolvePath(`test/extensions/${extensionId}/.contentfulrc.json`), {
+      cmaToken: cmaToken,
+      activeSpaceId: spaceId,
+      activeEnvironmentId: environmentId
+    })
+    await writeJSONFile(resolvePath('cypress.env.json'), {
```

Comment:
It should be written once. Not related to extension.

Reply:
I went through and made sure I wasn't rewriting anything else either.